src/GDGUkraine/templates/event.html

```diff
@@ -9,10 +9,21 @@
     <meta name="author" content="Olexandr Kovalchuk">
     <meta name="author" content="Svyatoslav Sydorenko">
     <meta name="author" content="Valentyn Shybanov">
+
+    <!-- Start opengraph meta -->
+    <!-- Override author, cause opengraph picks last author meta -->
+    <meta name="author" content="{{ event.host_gdg.name or 'GDG Ukraine' if event.host_gdg_id else 'GDG Ukraine' }}">
```

Comment:
Hm. I thought automatic escaping was enabled for project. I was wrong.

fixed in a54ad36 and 15ecdbe